owncloudApp/src/main/res/values-ja-rJP/strings.xml

```diff
@@ -369,9 +368,7 @@
   <string name="log_send_mail_subject">%1$s アンドロイドアプリログ</string>
   <string name="log_progress_dialog_text">読込中 ...</string>
   <string name="actionbar_select_all">すべて選択</string>
-  <string name="saml_authentication_required_text">認証を必要とする</string>
-  <string name="saml_authentication_wrong_pass">パスワードが間違っています</string>
-  <string name="actionbar_move">移動</string>
+    <string name="actionbar_move">移動</string>
```

Comment:
Same here

Reply:
Reason in https://github.com/owncloud/android/pull/2647#discussion_r319023746